### PR TITLE
Added specific pipes property constants

### PR DESCRIPTION
**Pull request by navinrathore**

### CODE REVIEW

**Comment by navinrathore** [comment]
* takes care of a part of https://github.com/zinggAI/zingg/issues/347

File: python/zingg/pipes/pipes.py [hunk 1]
```diff
@@ -6,46 +6,48 @@
...
+        Pipe.addProperty(self, FilePipe.DELIMITER, delimiter)
     def setLocation(self, location):
         Pipe.addProperty(self, FilePipe.LOCATION, location)
 
 class BigQueryPipe(Pipe):
     def __init__(self,name):
-        Pipe.__init__(self, name, "bigquery")
-        Pipe.addProperty(self, "viewsEnabled", "true")
+        Pipe.__init__(self, name, Format.BIGQUERY.type())
+        Pipe.addProperty(self, JBigQueryPipe.VIEWS_ENABLED, "true")
```
Comment (sonalgoyal) [inline python/zingg/pipes/pipes.py:24]: can we add this property by default?

File: python/zingg/pipes/pipes.py [hunk 2]
```diff
@@ -6,46 +6,48 @@
 
 Format = jvm.zingg.client.pipe.Format
 FilePipe = jvm.zingg.client.pipe.FilePipe
+JSnowflakePipe = jvm.zingg.client.pipe.SnowflakePipe
+JBigQueryPipe = jvm.zingg.client.pipe.BigQueryPipe
 
 class CsvPipe(Pipe):
     def __init__(self, name):
         Pipe.__init__(self, name, Format.CSV.type())
         Pipe.addProperty(self, FilePipe.HEADER,"true")
```
Comment (sonalgoyal) [inline python/zingg/pipes/pipes.py:15]: this is wrong - we can not set header default true. we have to let the user set the values.

**Review by sonalgoyal — CHANGES_REQUESTED** [review]

File: python/zingg/pipes/pipes.py (hunk 2 above)
Comment (navinrathore) [reply python/zingg/pipes/pipes.py:15]: I've removed it now.

File: python/zingg/pipes/pipes.py (hunk 1 above)
Comment (navinrathore) [reply python/zingg/pipes/pipes.py:24]: In Java app, there are no settings for BigQueryPipe,

**Comment by navinrathore** [comment]
> please add newlines between python methods.
Already in the code base
https://github.com/zinggAI/zingg/blob/main/python/zingg/pipes/pipes.py
through the below PR (for same comment here) 
https://github.com/zinggAI/zingg/pull/376#discussion_r913627667

File: python/zingg/pipes/pipes.py (hunk 1 above)
Comment (navinrathore) [reply python/zingg/pipes/pipes.py]: Property is no more set here.

**Pull request merged**